src/Context/Context.php

```diff
@@ -91,9 +92,15 @@ public function activate(): ScopeInterface
 
     private static function debugScopesDisabled(): bool
     {
-        $disabled = $_SERVER['OTEL_PHP_DEBUG_SCOPES_DISABLED'] ?? ini_get('OTEL_PHP_DEBUG_SCOPES_DISABLED');
+        if (isset($_SERVER[self::OTEL_PHP_DEBUG_SCOPES_DISABLED]) && filter_var($_SERVER[self::OTEL_PHP_DEBUG_SCOPES_DISABLED], FILTER_VALIDATE_BOOLEAN)) {
+            return true;
```

Comment:
I think that this means that if `OTEL_PHP_DEBUG_SCOPES_DISABLED` is `false` in any source, it will then look to the next entries. But, if it's actually set as false (as opposed to not set at all), the user presumably means they don't want it disabled?